x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/definition.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { entityDefinitionSchema, type EntityDefinition } from '@kbn/entities-schema';
+
+export const HOST_ENTITY_DEFINITION: EntityDefinition = entityDefinitionSchema.parse({
+  id: 'ea_host_entity_store',
+  name: 'EA Host Store',
+  type: 'node',
+  indexPatterns: ['log*'],
```

Comment:
Should we get this value from a constant? Like `DEFAULT_INDEX_PATTERN`?

Reply:
yeah, will do.
Just to clarify, do you mean a _new_ constant or re-use one of the more common ones? I'm still unclear on how exactly this is going to evolve, not to mention that this field can be overridden in `INIT engine` endpoint.

I think I';; just create an entity store/engine specific constant for now